guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,675 @@
+{
```

Comment:
<p>ArcGIS Hub allows you to build configurable and branded websites for sharing internal and public content, such as apps, data, and media. If you're using ArcGIS Hub Premium, you can launch a site by creating an <strong>initiative</strong> (insert premium hub link here). You can build a Hub Basic Site using the ArcGIS API for Python as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
What url is replacing _insert premium hub link here?_